client/lib/wp/sync-handler/README.md

```diff
@@ -19,3 +19,25 @@ const handler = new SyncHandler( wpcomXHRHandler );
 // create wpcom instance passing the wrapped handler
 const wpcom = wpcomUndocumented( handler );
 ```
+
+### Cache Invalidation
+
+`cache` attempts cleanup the obsolete records from the local storage of the browser so thus reduce the dB size. The current criteria are two:
```

Comment:
I'm not sure about the naming. I realize what we are doing is cache invalidation, but the module we are adding is not acting as a cache itself, `sync-handler/index.js` is the cache. So the name `cache` and the filename `cache.js` don't feel right to me.  What we're really creating here is an _index_, although that is obviously a problematic module name and filename in javascript. What about `cache-index` as a name? That would make sense to me for the method names as well: `cacheIndex.addItem( key )`, `cacheIndex.removeItem( key )`, etc.

If you agree, I would change this to:

```
### Cache Invalidation

`cache-index` stores the key and timestamp for all records stored by the `sync-handler` which allows us to prune records from our local cache. There are two methods to invalidate records:
```